ArdusatSDK.cpp

```diff
@@ -1,35 +1,52 @@
 /**
  * @file   ArdusatSDK.cpp
- * @Author Ben Peters (ben@ardusat.com)
+ * @author Ben Peters (ben@ardusat.com)
+ * @author Sam Olds (sam@ardusat.com)
  * @date   December 3, 2014
  * @brief  Implements ArdusatSDK generic sensor reading and configuration for Space Kit Sensors.
+ *
+ * Provides a unifying wrapper of sensor specific functionality to provide a
+ * consistent interface to interact with each type of sensor.
  */
 
 #include <stdio.h>
 #include <string.h>
 #include "ArdusatSDK.h"
 
-bool ARDUSAT_SHIELD = false;
-int OUTPUT_BUF_SIZE = 256;
+boolean MANUAL_CONFIG = false;
+boolean ARDUSAT_SPACEBOARD = false;
+int OUTPUT_BUF_SIZE = 512;
```

Comment:
`len("timestamp(millis),AccelerationX(m/s^2),AccelerationY(m/s^2),AccelerationZ(m/s^2),AmbientTemp(C),GyroX(rad/s),GyroY(rad/s),GyroZ(rad/s),IRTemp(C),Luminosity(lux),MagneticX(uT),MagneticY(uT),MagneticZ(uT),Pressure(hPa),RGBLightRed(lux),RGBLightGreen(lux),RGBLightBlue(lux),RGBLightRed(lux),RGBLightGreen(lux),RGBLightBlue(lux),UVLight(mW/cm^2),UVLight(mW/cm^2),checksum")` = 367

So I'll switch the buffer size back to 256 and then just require the CSV writer to get access to the `serialConnection`.